lib/purescript-cst/package.yaml

```diff
@@ -0,0 +1,77 @@
+name: purescript-cst
+version: '0.13.6'
+synopsis: PureScript Programming Language Concrete Syntax Tree
+description: The surface syntax of the PureScript Programming Language.
+category: Language
+author: Phil Freeman <paf31@cantab.net>
+maintainer: >
+  Gary Burgess <gary.burgess@gmail.com>,
+  Hardy Jones <jones3.hardy@gmail.com>,
+  Harry Garrood <harry@garrood.me>,
+  Christoph Hegemann <christoph.hegemann1337@gmail.com>,
+  Liam Goodacre <goodacre.liam@gmail.com>,
+  Nathan Faubion <nathan@n-son.com>
+copyright: (c) 2013-17 Phil Freeman, (c) 2014-19 Gary Burgess, (c) other contributors (see CONTRIBUTORS.md)
+license: BSD3
+github: purescript/purescript
+homepage: http://www.purescript.org/
+extra-source-files:
+  - tests/purs/layout/*.purs
+  - README.md
+dependencies:
+  - array
+  - base >=4.11 && <4.13
+  - containers
+  - dlist
+  - purescript-ast
+  - scientific >=0.3.4.9 && <0.4
+  - semigroups >=0.16.2 && <0.19
+  - text
+build-tools:
+  - happy ==1.19.9
+
+library:
```

Comment:
Not sure where to put this, so I'll arbitrarily choose this line.

I want to call attention to the commit message:

> It's important to note that we don't pull out `Language.PureScript.CST`.
That module is straddling the line between a re-exporting "prelude" and
an adapter between `purescript-ast` and `purescript`. Since it deals
with errors, and errors currently exist at the top-level, we cannot yet
move this module down into `purescript-cst`.
>
> Probably what needs to happen is that we make a split between
`Language.PureScript.CST` being a re-exporter and
`Language.PureScript.CST` doing adapter work. Not clear which way to go
on this one as both seem viable avenues.

I think this is an important point to make. Just because the module has `CST` in it, doesn't mean it has to go with the `purescript-cst` package. That module is doing double duty. If we wanted to separate the responsibilities, we could do that. Though, I think the re-exporting is an anti-pattern for this project. If we wanted to rename it, we could do that. If we don't care, we can leave it. I don't have any strong feelings, but wanted to make sure this was explicitly known about.